### ISSUE TITLE
bug: Failed to execute 'getComputedStyle' on 'Window'

### ISSUE THREAD

**Issue by RangerRick** [driftyco/ionic#3182]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>all</span>  

<span ionic-description>I get the following error in my code using `collection-repeat` in the latest nightlies (it does not happen with `ng-repeat`):

```
TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'. ionic.bundle.js:19516
    at TypeError (native)
    at RepeatController.refreshLayout (http://code.ionicframework.com/nightly/js/ionic.bundle.js:49656:55)
    at HTMLElement.refreshDimensions (http://code.ionicframework.com/nightly/js/ionic.bundle.js:49489:21)
    at http://code.ionicframework.com/nightly/js/ionic.bundle.js:11147:14
    at forEach (http://code.ionicframework.com/nightly/js/ionic.bundle.js:8232:20)
    at forEach.triggerHandler (http://code.ionicframework.com/nightly/js/ionic.bundle.js:11145:7)
    at Object.JQLite.(anonymous function) [as triggerHandler] (http://code.ionicframework.com/nightly/js/ionic.bundle.js:11161:17)
    at http://code.ionicframework.com/nightly/js/ionic.bundle.js:47904:14
    at http://code.ionicframework.com/nightly/js/ionic.bundle.js:24132:28
    at completeOutstandingRequest (http://code.ionicframework.com/nightly/js/ionic.bundle.js:12814:10)
 (anonymous function) ionic.bundle.js:19516
 $get ionic.bundle.js:16466
 (anonymous function) ionic.bundle.js:24135
 completeOutstandingRequest ionic.bundle.js:12814
 (anonymous function) ionic.bundle.js:13194
Uncaught TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'. ionic.bundle.js:49656
 refreshLayout ionic.bundle.js:49656
 refreshDimensions ionic.bundle.js:49489
```

I've seen it in Chrome and the iOS emulator.

Reproducible in this codepen: http://codepen.io/RangerRick/full/MYXgeO</span>

<span is-issue-template></span>

**Comment by ajoslin** [comment]
Thanks, checking this out.

**Comment by ajoslin** [comment]
Not getting this in the latest nightly.  http://codepen.io/ionic/pen/RNJRzQ

I added something to make it so there are 1000 alarms.


```  js
  // Create 100 copies of alarm
  var alarm = $scope.alarms[0];
  for (var i = 0; i< 100; i++) 
    $scope.alarms.push(angular.extend({}, alarm));
```

**Comment by RangerRick** [comment]
Confirmed, this doesn't happen in my app with the latest nightly.

**Comment by YayConnolly** [comment]
I am getting this error, in my code and when I visit your codepen @ajoslin

**Comment by ajoslin** [comment]
Not getting this in the codepen @YayConnolly.

**Comment by YayConnolly** [comment]
Dang you respond fast @ajoslin. Here's a screenshot, tried clearing out cached browser data:

![screenshot](https://cloud.githubusercontent.com/assets/1444054/6450366/2100416a-c0dc-11e4-9f0b-ea26a6eeb5f5.png)

**Comment by RangerRick** [comment]
Hah, I stand corrected.  I just saw it again in my full app as well.  Perhaps it's a race condition of some kind, so it only shows intermittently?

**Comment by YayConnolly** [comment]
Appears to be working with the latest version in ionic-bower repo: v1.0.0-beta.14-nightly-1102. Will post back if this becomes an issue again. Thanks!

**Comment by ajoslin** [comment]
Fixed this earlier today via 873cded. I just forgot to reference this issue. Thanks for the reports, all.
